src/main/java/com/iota/iri/network/Node.java

```diff
@@ -107,7 +110,7 @@ public void init() throws Exception {
         P_SEND_MILESTONE = configuration.doubling(Configuration.DefaultConfSettings.P_SEND_MILESTONE.name());
         P_REPLY_RANDOM_TIP = configuration.doubling(Configuration.DefaultConfSettings.P_REPLY_RANDOM_TIP.name());
         P_PROPAGATE_REQUEST = configuration.doubling(Configuration.DefaultConfSettings.P_PROPAGATE_REQUEST.name());
-        sendLimit = (long) ((configuration.doubling(Configuration.DefaultConfSettings.SEND_LIMIT.name()) * 1000000) / (TRANSACTION_PACKET_SIZE * 8));
+        sendLimit = (long) ((configuration.doubling(Configuration.DefaultConfSettings.SEND_LIMIT.name()) * 1000000) / (configuration.integer(Configuration.DefaultConfSettings.TRANSACTION_PACKET_SIZE) * 8));
```

Comment:
you can use `packetSize` instead of reparsing the `TRANSACTION_PACKET_SIZE` conf